pkg/hs/urbit-king/lib/Urbit/Vere/Term.hs

```diff
@@ -175,14 +175,28 @@ _spin_idle_us = 500000
 -}
 localClient :: ∀e. HasLogFunc e
             => STM ()
-            -> RAcquire e (TermSize, Client)
+            -> RAcquire e Client
 localClient doneSignal = fst <$> mkRAcquire start stop
   where
-    start :: HasLogFunc e => RIO e ((TermSize, Client), Private)
+    start :: HasLogFunc e => RIO e (Client, Private)
     start = do
       tsWriteQueue  <- newTQueueIO :: RIO e (TQueue [Term.Ev])
       spinnerMVar   <- newEmptyTMVarIO :: RIO e (TMVar ())
-      tsizeTVar     <- io $ T.termSize >>= newTVarIO
+
+      -- Track the terminal size, keeping track of the size of the local
+      -- terminal for our own printing, as well as putting size changes into an
+      -- event queue so we can send changes to the terminal muxing system.
+      tsizeTVar         <- newTVarIO (TermSize 80 24) -- Value doesn't matter.
+      tsSizeChangeQueue <- newTQueueIO
```

Comment:
Why do we need a queue for this, rather than a TMVar (bounded queue of size one)?

Reply:
Done.